src/terminal/parser/InputStateMachineEngine.cpp

```diff
@@ -965,6 +1145,12 @@ _Success_(return ) bool InputStateMachineEngine::_GetXYPosition(_In_reads_(cPara
         *puiLine = rgusParams[0];
         *puiColumn = rgusParams[1];
     }
+    else if (cParams == 3)

```

Comment:
I don't love overloading this method to also get the XY position from SGR sequences. I think I'd prefer a unique method for getting the XY position from an SGR mouse sequence, that only accepts 3 params.

Reply:
I'll do it. I just don't understand why. So, why?

Reply:
wait. I get it now. nvmd